### ISSUE TITLE
Missing snappy on helm repository

### ISSUE THREAD

**Issue by GiuseppeP** [vernemq/docker-vernemq#207]
Hello,
I'm getting error due missing of libsnappy.so.1 .
I found it was fixed in **1.10.2-1**, but after almost 2 months it wasn't added in helm repository.

latest-apline resolve to:
`    Image:          vernemq/vernemq:1.10.2-alpine`

Any other way I can resolve  waiting for the update of helm repo?

Thanks,

**Comment by NULLx76** [comment]
Specifying the exact image in the `values.yaml` worked for me:
```yml
image:
  repository: vernemq/vernemq
  tag: 1.10.2-1-alpine
```

**Comment by GiuseppeP** [comment]
Yes it worked for me too, I used that way as temp solution, but I want keep it updated to latest version, instead of forcing installation of a specific version.

**Comment by codeadict** [comment]
This is fixed now, both 1.10.2-1 and 1.10.3 are released on the helm repo.
